### ISSUE TITLE
Set Up a DB to store scrapped posts...etc.

### ISSUE THREAD

**Issue by Tdarra** [OpenSOROS/OpenSOROS#4]
@mariocimet we're gonna need to set up some sort of database of our findings to archive at least. That would be a good first step before modelling and analysis I think.

**Comment by mariocimet** [comment]
Yes, you're right. What would you suggest? 

I have $100 AWS credit from a Hackathon but never got around to using it. I've used Google Cloud in the past and found it easy to work with. Either way I have very little experience setting up a database.

Another option while the data is small is to just use something like SQL Alchemy and store the data locally or on Github, right?

Thoughts? @Tdarra  @arturenault

**Comment by arturenault** [comment]
I can look into if I can get any discounts with Azure and what not

**Comment by Tdarra** [comment]
I can get discounts using compose (IBM) they have some solid open source db's on there we could use. https://www.compose.com/

**Comment by mariocimet** [comment]
nice, will check it out

**Comment by Tdarra** [comment]
We could also totally use that credit since we won't have crazy amounts of data. We can use things like Hadoop and Spark if we hit up Amazon too. Otherwise if we wanna go local I have a linux distro lying around on some bare metal we could set up as our server/db?

**Comment by mariocimet** [comment]
@Tdarra I'm convinced. Lets use my credit for now, I'll get that set up. 

These comments about scale of data and the price of hosting/analyzing the data make me think we should plan a sprint, something neatly encapsulated.

**Comment by mberrett** [comment]
Google Cloud is fun and cool, but costs can sneak up on you, although you do get a free year trial starting with $300

**Comment by mariocimet** [comment]
Aight, I got everything set up to get my credits - now comes the question what kind of database do we want?

I'm thinking about the choice between Amazon S3, and the SQL/NoSQL options. A quick read online tells me that you want S3 for slowish access to comparatively few, larger objects (you have unlimited object size), while the databases are for very fast access to very large numbers of small objects (max object size is something like 65kb)

S3 seems to me to be the right option here, but I'm a DB n00b and would appreciate guidance from those more in the know @Tdarra @arturenault

**Comment by mariocimet** [comment]
Maybe we put this on hold and investigate whats already available? I'm thinking things like https://www.reddit.com/r/datasets/comments/3bxlg7/i_have_every_publicly_available_reddit_comment/ that were referenced in #1

**Comment by mariocimet** [comment]
@yp1996 this issue deals with the data-warehousing issues we were talking about earlier. 

Teddy Groves expressed interest in throwing the MLA twitter data into LIWC, could we provide him with csvs of the raw twitter data? (would be a little more user friendly than telling him to just clone and run the scripts, for instance)

**Comment by yp1996** [comment]
@mariocimet yeah the raw data is in our BC election google drive folder actually [(link)](https://drive.google.com/file/d/0B-pWLAo2HpWqS3hJZEtRblVUYXc/view?usp=sharing)
